### PR TITLE
fastdmg 1.0.2 (new cask)

### LINKED ISSUE #6: Installation via `brew`
Thank you for such an awesome little utility!

It would be nice to be able to install it via `brew`.

### PR DESCRIPTION
**Pull request by Moulick**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---
closes https://github.com/sveinbjornt/FastDMG/issues/6

### CODE REVIEW

**Review by khipp — COMMENTED** [review]
Thank you, @Moulick! I left a few suggestions below.

File: Casks/f/fastdmg.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+cask "fastdmg" do
+  version "1.0.2"
+  sha256 :no_check
+
+  url "https://sveinbjorn.org/files/software/FastDMG.zip"
```
[the remark this answers is not in the document]
Comment (Moulick) [reply Casks/f/fastdmg.rb]: Neat, I got the URL from the GitHub readme and it was unversioned. Apparently the homepage _has_ a versioned URL, Nice!

**Comment by Moulick** [comment]
Thanks @khipp for your fixes, all applied :)

File: Casks/f/fastdmg.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+cask "fastdmg" do
+  version "1.0.2"
+  sha256 "405c99ee251ebe1a5f3e4cbec8b84e6211ee8b89e5abf0f50b7b2cee9567062a"
+
+  url "https://sveinbjorn.org/files/software/fastdmg/FastDMG-#{version}.zip"
+  name "FastDMG"
+  desc "Replacement for Apple's DiskImageMounter app"
```
Comment (bevanjkay) [inline Casks/f/fastdmg.rb:7]:
```suggestion
  desc "Alternative to Apple's DiskImageMounter app"
```

Minor nit, but "replacement" reads to me like Apple's option no longer exists.

**Review by bevanjkay — APPROVED** [review]

File: Casks/f/fastdmg.rb (hunk 2 above)
Comment (Moulick) [reply Casks/f/fastdmg.rb:7]: Can we write "Faster alternative to Apple's DiskImageMounter app"? As that's not like marketing fluffbut the whole reason this exist in the first place.

Comment (bevanjkay) [reply Casks/f/fastdmg.rb:7]: We tend not to because statements like "faster" can be a bit circumstantial and may not stay true forever.

Comment (Moulick) [reply Casks/f/fastdmg.rb]: Ok, Nit commited, now need a new approver :(

**Review by bevanjkay — APPROVED** [review]

File: Casks/f/fastdmg.rb (hunk 2 above)
Comment (bevanjkay) [reply Casks/f/fastdmg.rb]: I'm still here, thanks @Moulick

Comment (Moulick) [reply Casks/f/fastdmg.rb]: Hah. I assumed that if I took your code suggestion and as a commit now shows up as co-authored by me + you, GitHub would not accept your approval. But maybe `homebrew-cask` has it allowed in the branch ruleset(s)?

**Pull request merged**